schema/ListedLicense.xsd

```diff
@@ -39,12 +39,12 @@
 		<choice minOccurs="1" maxOccurs="unbounded">
 			<element name="crossRefs" type="tns:crossRefsType" minOccurs="0" maxOccurs="1"/>
 			<element name="notes" type="string" minOccurs="0" maxOccurs="1"/>
-			<element name="standardLicenseHeader" type="tns:formattedTextType" minOccurs="0" maxOccurs="1"/>
-			<element name="titleText" type="tns:formattedTextType" minOccurs="0" maxOccurs="1"/>
-			<element name="copyrightText" type="tns:formattedTextType" minOccurs="0" maxOccurs="1"/>
+			<element name="standardLicenseHeader" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1"/>
```

Comment:
I believe we decided not to support optional and alt tags in the license header for this release.  That being said, I could not find any record of that decision (gave up after a 10 minute search).

Reply:
> I believe we decided not to support optional and alt tags in the license header for this release.

But thing like `ISC` have `<alt>` and `<optional>` and are the standard headers (although we don't say so yet, see #451).  Is there a techical problem with them?  Once we have alt/optional matching for the licenses, couldn't we use it for headers too?

Reply:
@wking Are there any existing licenses using <alt> and/or <optional> in the standard headers?  If not, we can implement this in the next release of the license list.

Reply:
> Are there any existing licenses using and/or in the standard headers?

Yup:

```
$ grep -rlz '<standardLicenseHeader>.*\(<alt \|<optional>\).*</standardLicenseHeader>' src | sort
src/AGPL-3.0.xml
src/Apache-2.0.xml
src/CPAL-1.0.xml
src/CUA-OPL-1.0.xml
src/ECL-1.0.xml
src/ECL-2.0.xml
src/GFDL-1.1.xml
src/GFDL-1.2.xml
src/GFDL-1.3.xml
src/GPL-1.0.xml
src/GPL-2.0.xml
src/GPL-3.0.xml
src/Interbase-1.0.xml
src/LGPL-2.0.xml
src/LGPL-2.1.xml
src/LPPL-1.0.xml
src/LPPL-1.1.xml
src/LPPL-1.2.xml
src/LPPL-1.3a.xml
src/LPPL-1.3c.xml
src/MPL-1.0.xml
src/MPL-1.1.xml
src/OCLC-2.0.xml
src/RPSL-1.0.xml
src/SGI-B-1.0.xml
src/SGI-B-1.1.xml
src/SISSL-1.2.xml
src/SISSL.xml
src/SPL-1.0.xml
src/W3C-20150513.xml
src/W3C.xml
```

Reply:
OK - sounds like we should support the markup in this release.   I logged 2 separate issues for the SPDX tools to support templates in the standard license header.  I do not think either of these issues are critical to releasing this version of the license list, so we can proceed with keeping this in the schema.

Issues logged: 

* Support templates for the standard license header in the library model: https://github.com/spdx/tools/issues/116
* Highlight the alt and optional text in the standard license headers: https://github.com/spdx/tools/issues/115